packages/components/src/components/input-range/component.tsx

```diff
@@ -83,6 +94,14 @@ export class KolInputRange implements API {
 		}
 	};
 
+	componentDidLoad() {
+		if (this.refInputange) {
+			if (!this._value) {
+				this._value = parseFloat(this.refInputange.value);
```

Comment:
(Warum) Benötigen wir das zusätzlich zu Zeile 52, in der wir den Wert bereits in den State übernehmen?